snakemake/ioutils/input.py

```
@@ -0,0 +1,27 @@
+            if parser is None:
+                return fh.read().strip()
+            else:
+                return parser(fh, **kwargs)
+
+    return inner
+
+
+def extract_checksum(infile, **kwargs):
+    import pandas as pd
+
+    fix_file_name = lambda x: x.removeprefix("./")
```

Ruff (0.8.2)

Do not assign a `lambda` expression, use a `def`

Rewrite `fix_file_name` as a `def`

(E731)